.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 dist/js/i18n/build.txt
+.idea
```

Comment:
These should be ignored in your user-level `.gitignore` (the `~/.gitignore` file), not in the project-level one.